schedulingApp/models.py

```diff
@@ -8,15 +8,65 @@ class User(models.Model):
         ('admin', 'Admin')
     )
 
-    emailAddress = models.CharField(max_length=32)
-    phoneNumber = models.CharField(max_length=16)
-    homeAddress = models.CharField(max_length=64)
     firstName = models.CharField(max_length=32)
     lastName = models.CharField(max_length=32)
+    password = models.CharField(max_length=32)
+    emailAddress = models.CharField(max_length=32)
+    homeAddress = models.CharField(max_length=64)
+    phoneNumber = models.CharField(max_length=16)
     permission = models.CharField(max_length=16,
                                   choices=PermissionLevel,
                                   default=PermissionLevel[0])
 
+    # constructor/destructor
+    def __init__(self, firstName, lastName, password, emailAddress, homeAddress, phoneNumber, permission):
+        pass
+
+    def __del__(self):
+        pass
+
+    # getters/setters
+    def getFirstName(self):
```

Comment:
You can do stuff like User.firstName or User.firstName = "123", but since user.profile.emailAddress must be used and for validation against invalid phone numbers/emails, looks fine to me to have these functions.